lib/widgets/action_sheet.dart

```diff
@@ -146,9 +107,10 @@ abstract class MessageActionSheetMenuItemButton extends StatelessWidget {
         padding: const EdgeInsets.symmetric(vertical: 12, horizontal: 16),
         foregroundColor: designVariables.contextMenuItemText,
         splashFactory: NoSplash.splashFactory,
-      ).copyWith(backgroundColor: WidgetStateColor.resolveWith((states) =>
-          designVariables.contextMenuItemBg.withFadedAlpha(
-            states.contains(WidgetState.pressed) ? 0.20 : 0.12))),
+      ).copyWith(backgroundColor: WidgetStateColor.fromMap({
+        WidgetState.pressed: designVariables.contextMenuCancelPressedBg,
+        ~WidgetState.pressed: designVariables.contextMenuCancelBg,
```

Comment:
This seems wrong — it's using the cancel button's colors for the normal buttons.

(Noticed this in debugging with @rajveermalviya why the rebase of #1103 atop these changes had a color mismatch between these buttons and that PR's new ones.)